dev/openchpl_views.sql

```diff
@@ -1102,6 +1106,10 @@ CREATE VIEW openchpl.certified_product_summary AS
     cp.last_modified_user,
     cp.deleted,
     cp.pending_certified_product_id,
+    cp.rwt_plan_url,
```

Comment:
I don't think this view has the other/original rwt column rwt_eligibility_year. Does it need that one too (like the details view has)?